src/.nuget/NuGetRestore.ps1

```diff
@@ -1,5 +1,6 @@
 $NuGetExe = "$PSScriptRoot\NuGet.exe"
 
+& $NuGetExe install xunit.runner.console -version 2.0.0 -OutputDirectory "$PSScriptRoot\..\..\packages"
```

Comment:
@tannergooding I'm not sure I understand what you're asking.

The tests run fine with the 2.0.0 version of xUnit. The analyzer unit tests, as compared to some of the other Roslyn unit tests, are very vanilla. We do still run into the problem where fake-signed assemblies won't run when shadow-copied, but I just turn off shadow copying everywhere we invoke the console runner.

xunit.runner.visualstudio does *not* work because it does not provide a mechanism for turning off shadow-copying, as far as I can tell.